src/main/resources/assets/wotr/models/block/key_forge.json

```diff
@@ -1,306 +1,257 @@
 {
-	"parent": "block/block",
```

Comment:
I would recommend adding the parent line to the model (this is what ensures the block appearance in inventory has the isometric angle instead of side-on).